server/data-migrations/1654596810662-ConvertAllUserEmailsToLowercaseAndDeleteDuplicateUsers.ts

```diff
@@ -24,50 +26,34 @@ export class ConvertAllUserEmailsToLowercaseAndDeleteDuplicateUsers1654596810662
       .createQueryBuilder('users')
       .select(['users.id', 'users.email', 'users.invitationToken'])
       .leftJoinAndSelect('users.organizationUsers', 'organizationUsers')
-      .leftJoinAndSelect('users.apps', 'apps')
```

Comment:
`@Afterload` query for app versions was being triggered for each rows selected from the app entity and thus resulting in timeouts when running migrations